pyomo/core/base/var.py

```diff
@@ -1010,13 +1010,27 @@ def domain(self):
     @domain.setter
     def domain(self, domain):
         """Sets the domain for all variables in this container."""
-        # TODO: Ideally we would pass valid arguments to the initializer
```

Comment:
I cannot express emphatically enough how excited I am that a TODO was _deleted_.